pkgs/applications/networking/instant-messengers/signald/default.nix

```diff
@@ -138,6 +84,6 @@ in stdenv.mkDerivation {
     ];
     license = licenses.gpl3Plus;
     maintainers = with maintainers; [ expipiplus1 ma27 ];
-    platforms = [ "x86_64-linux" "aarch64-linux" ];
+    platforms = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
This change seems unrelated.

Reply:
it's related to lines 66-69. Previously, two sets of deps were maintained - one for linux-aarch64, one for linux-x86_64. Presumably, Darwin wasn't supported because the author didn't have such a builder or didn't want to maintain more hashes. My changes allow all 4 sets of deps (`{linux,darwin}-{x86_64,aarch64}`) to be stored in one deps.json, so the supported platforms list can now be extended. Of course, if it turns out it can't *actually* be built on those platforms, this can be reverted.

Reply:
We should check that before merging (I can do `aarch64-darwin` these days, but not much more than a smoke-test), but then I'd consider this thread to be solved.

Reply:
It builds and runs, so I'd say this can be closed. Even if there will be some issues, it will have to be set as `broken` rather than removed from `platforms`.